### ISSUE TITLE
weird behaviour with save/load workspaces

### ISSUE THREAD

**Issue by iasj** [szw/vim-ctrlspace#125]
I'm using vim-ctrlspace (it is wonderful by the way) and I recently noticed a weird behaviour related with workspaces. I'll try to explain. I have this C-project structure:

```console
dev  ▶ tree
.
├── bin
│   └── Makefile
├── doc
│   ├── erros.md
│   └── questions.md
├── Makefile
├── README.md
└── src
    ├── complex.c
    ├── complex.h
    ├── list.c
    ├── list.h
    ├── main.c
    ├── Makefile
    ├── math.h
    ├── matrix.c
    ├── matrix.h
    ├── node.c
    └── node.h

3 directories, 16 files
```

and this project will grow exponetially very soon. So, I created two workspaces: "Code" and "Doc". To create them I did the follow:

1. Opened vim-ctrlspace window
2. Press 'N' to create a blank workspace
3. Opened vim-ctrlspace window
4. Press 'w' to enter in workspaces window
5. Press 's' to toggle to save mode ([∷])
6. Press 'tab' to choose a name and save the new workspace

after that I started to fill my new workspaces with a lot of file disposed with tabs of windows containing more than one file, splited in each window. 

The problem is that vim-ctrlspace is messing with the predefined layout that I already saved. In the first, tab of the workspace "Code", I have the files "src/main.c" and the 3 Makefiles splited, and then I use [vim dwm][dwm] to easyly switch between them. Here is screenshots before and after:

* Before (When I save the workspace with :CtrlSpaceSaveWorkspace)
![vim-ctrlspace - before](https://cloud.githubusercontent.com/assets/7888397/9088849/8cf3b3fa-3b68-11e5-9fcc-cc9a4ac01b2f.png)

* After (When I Load the workspace "Code")
![vim-ctrlspace - after](https://cloud.githubusercontent.com/assets/7888397/9088908/e5b6a380-3b68-11e5-8c92-c93bd96f6457.png)

you can see that "bin/Makefile" is not opened anymore. And worst, the "src/main.c" was opened twice. When I load the workspace again, "src/main.c" is opened again (3 of them), it's like a virus replication. This is happening in other tabs too, and it is driving me crazy!

Here is my configuration:

```vimscript
" For br-abnt2 keyboard layout. This key is in 
" the same position as sime-colon (;) is in others 
" layouts, i.e. right beside 'l' key. Is far for 
" pratical than Ctrl+Space, because ctrl key is 
" far away from " my thumb.
let g:ctrlspace_default_mapping_key = 'ç'

let g:ctrlspace_height = 1
let g:ctrlspace_max_height = 10
let g:ctrlspace_save_workspace_on_exit = 1
let g:ctrlspace_save_workspace_on_switch = 1
let g:ctrlspace_load_last_workspace_on_start = 0
"let g:ctrlspace_cache_dir = "~/.vim"
let g:ctrlspace_max_files = 100
let g:ctrlspace_max_search_results = 100
let g:ctrlspace_use_ruby_bindings = 1
let g:ctrlspace_use_tabline = 1
set wildignore=*.o,*.obj,*.png,*.jpg,*.jpeg
```

please, any help will be appreciated.
[dwm]: https://github.com/spolu/dwm.vim

**Comment by szw** [comment]
Hi! 

I've checked your scenario briefly (tried to reproduce) and in Vim-CtrlSpace 5.0 everything seems to work. Could you please check it with the new [Version 5](https://github.com/szw/vim-ctrlspace/pull/124) as version 4.0 is frozen and this one will be merged soon. 
Additionally, please check if the problem occurs in vanilla Vim (e.g. leave only Vim-Ctrlspace and Vundle plugins). 

Cheers!
s.

**Comment by iasj** [comment]
Hi! Firstly, thanks for the answer so quickly. I tried what you said. 

I started Vim with only vim-ctrlspace, and all worked fine. I even removed cs_workspaces, inside the .git, just in case. 

I descovered that [dwm plugin][dwm] is, some how, in conflit with vim-ctrlspace. I don't know how yet. This is a sad thing, because I use them to make the perfect workflow. 

I'll try to figure what is happening, but until I don't I think I'll have to choose one of them (trick choice).

I'll appreciate any clue about them, anything at all!

[dwm]: https://github.com/spolu/dwm.vim
